apps/web/pages/v2/event-types/index.tsx

```diff
@@ -317,7 +317,6 @@ export const EventTypeList = ({ group, groupIndex, readOnly, types }: EventTypeL
                             <DropdownMenuContent>
                               <DropdownMenuItem>
                                 <DropdownItem
-                                  type="button"
```

Comment:
type = button tells browser to not submit the containing form. So, this is required.